contrib/charts/cert-manager/README.md

```diff
@@ -12,17 +12,25 @@ to renew certificates at an appropriate time before expiry.
 
 ## Installing the Chart
 
-Full installation instructions, including details on how to configure extra
-functionality in cert-manager can be found in the [official deploying docs](https://github.com/jetstack/cert-manager/blob/master/docs/user-guides/deploying.md#addendum).
+Due to a [bug](https://github.com/kubernetes/helm/issues/2994#issuecomment-357844513)
+in helm, we can't install CRDs and their definition in the same step.
+
+First, install the chart with the CRDs and then install the default
+ClusterIssuer by specifying your letsencryptEmail value.
 
 To install the chart with the release name `my-release`:
 
 ```console
 $ helm install --name my-release stable/cert-manager
 ```
 
-In order to begin issuing certificates, you will need to set up a ClusterIssuer
-or Issuer resource (for example, by creating a 'letsencrypt-staging' issuer).
+During installation step, no issuers is created. As an example, and to help
+you get started, we included 2 ClusterIssuers for Let's encrypt staging and
+production. To activate them, you need to upgrade the chart and give your email address:
```

Comment:
I think we should leave the old text as is, and instead tack this as another paragraph starting "The cert-manager helm chart can optionally be used to deploy a default set of Issuer resources ..." (and then go on to explain the nuances around install vs upgrade etc). Otherwise we risk making users think the only way to manage or create Issuer resources is via the Helm chart.

My real fear with this, is we will end out needing to expose the entire surface of the `Issuer` resource type via the values.yaml unless we make it clear that this is for convenience only, and not really a best practice. For example, as soon as we merge this, I would guess within a couple of weeks we'll have someone open an issue asking how to configure their DNS provider on the created Issuer resource. I *really* don't want to start exposing all this in values.yaml, as we don't have a strong strategy for versioning this manifest, thus as soon as we release a new version that changes the API spec, users will have to know to fix their existing values file (and helm's version system is not particularly clear from an end user perspective)